### PR TITLE
Enabled keeping the existing data in update-policy

### PR DESCRIPTION
**Pull request by guykeller**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17186

## Description
The update-policy command used to replace the existing data with the arguments supplied.
Now there is a possibility to keep the existing data and add / remove the supplied arguments to it.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by ronykoz** [comment]
@kirbles19 this is for the next release, could you check out the new arguments description please? + the RN

File: Integrations/integration-Fortigate.yml [hunk 1]
```diff
@@ -119,6 +119,26 @@ script:
...
+        if bool_to_str(keep_original_data):
+            policy_data = get_policy_request(policy_id)[0] # the return value is an array with one element
+            existing_adresses_list = policy_data.get(policy_field)
+            existing_adresses_list = [address_data["name"] for address_data in existing_adresses_list]
+            if add_or_remove.lower() == "add":
+                for address in existing_adresses_list:
+                    if not address in address_list_for_request:
+                        address_list_for_request.append(address)
+            else:
+                address_list_for_request = [address for address in existing_adresses_list if address not in address_list_for_request]
```
Comment (ronykoz) [inline Integrations/integration-Fortigate.yml]: how is it different from lines 133-135

File: Integrations/integration-Fortigate.yml [hunk 2]
```diff
@@ -119,6 +119,26 @@ script:
...
+        address_data_dicts_for_request = policy_addr_array_from_arg(address_list_for_request, False)
```
Comment (ronykoz) [inline Integrations/integration-Fortigate.yml]: are you making it a recursive call?

File: Integrations/integration-Fortigate.yml (hunk 1 above)
Comment (guykeller) [reply Integrations/integration-Fortigate.yml]: Look again, these are two opposite actions:
On lines 133-135, I add every address from the existing addresses to the arg addresses if it isn't there yet.
The purpose of these lines is to make sure the updated source/destination includes all the existing addresses + the new ones.
On the other hand, on line 137 I'm taking the existing addresses and removing all the arg addresses.

**Pull request merged**